packages/transformer/src/test/standalone/IModelTransformerHub.test.ts

```diff
@@ -4846,6 +4849,140 @@ describe("IModelTransformerHub", () => {
     await tearDown();
   });
 
+  it("should correctly handle processChanges when inserting LineStyles", async () => {
+    if (!semver.gte(coreBackendPkgJson.version, "4.10.12")) {
+      return; // Pre 4.10.12 does not properly import line styles, so test will fail in older versions
+    }
+    const sourceIModelName: string =
+      IModelTransformerTestUtils.generateUniqueName("Source");
+    const sourceIModelId = await HubWrappers.recreateIModel({
+      accessToken,
+      iTwinId,
+      iModelName: sourceIModelName,
+      noLocks: true,
+    });
+
+    const targetIModelName: string =
+      IModelTransformerTestUtils.generateUniqueName("Target");
+    const targetIModelId = await HubWrappers.recreateIModel({
+      accessToken,
+      iTwinId,
+      iModelName: targetIModelName,
+      noLocks: true,
+    });
+
+    // open/upgrade sourceDb
+    const sourceDb = await HubWrappers.downloadAndOpenBriefcase({
+      accessToken,
+      iTwinId,
+      iModelId: sourceIModelId,
+    });
+
+    const lineStyleStrokes: LineStyleDefinition.Strokes = [];
+    lineStyleStrokes.push({
+      length: 4,
+      strokeMode: LineStyleDefinition.StrokeMode.Dash,
+    });
+    lineStyleStrokes.push({ length: 2 });
+
+    const lineStyleProps =
+      LineStyleDefinition.Utils.createStrokePatternComponent(sourceDb, {
+        descr: "test line style",
+        strokes: lineStyleStrokes,
+      });
+
+    const lineStyle1Id = LineStyleDefinition.Utils.createStyle(
+      sourceDb,
+      IModelDb.dictionaryId,
+      "TestLineStyle",
+      lineStyleProps
+    );
+
+    sourceDb.saveChanges();
+    await sourceDb.pushChanges({
+      accessToken,
+      description: "First LineStyle insert",
+    });
+
+    const targetDb = await HubWrappers.downloadAndOpenBriefcase({
+      accessToken,
+      iTwinId,
+      iModelId: targetIModelId,
+    });
+    let transformer = new IModelTransformer(sourceDb, targetDb);
+    await transformer.process();
```

Comment:
saving changes is off , source changes should be saved before calling transformer. It should be something like:
```
    await transformer.process();
    transformer.dispose();
    targetDb.saveChanges();
    await targetDb.pushChanges({
      accessToken,
      description: "First LineStyle insert",
    });
```

Reply:
Yeah thanks, I added this yesterday as well after Vilius noticed it was missing. Just didn't commit the change yet